zeebe/bpmn-model/src/test/java/io/camunda/zeebe/model/bpmn/instance/zeebe/ZeebeCalledDecisionTest.java

```diff
@@ -37,6 +44,43 @@ public Collection<ChildElementAssumption> getChildElementAssumptions() {
   public Collection<AttributeAssumption> getAttributesAssumptions() {
     return Arrays.asList(
         new AttributeAssumption(BpmnModelConstants.ZEEBE_NS, "decisionId", false, true),
-        new AttributeAssumption(BpmnModelConstants.ZEEBE_NS, "resultVariable", false, true));
+        new AttributeAssumption(BpmnModelConstants.ZEEBE_NS, "resultVariable", false, true),
+        new AttributeAssumption(
+            BpmnModelConstants.ZEEBE_NS, "bindingType", false, false, ZeebeBindingType.latest));
+  }
+
+  @Test
+  public void shouldReadValidBindingType() {
+    // given
+    final BpmnModelInstance modelInstance =
+        Bpmn.readModelFromStream(getClass().getResourceAsStream("ZeebeCalledDecisionTest.bpmn"));
+    final BusinessRuleTask businessRuleTask =
+        modelInstance.getModelElementById("business-rule-task-1");
+
+    // when
+    final ZeebeCalledDecision calledDecision =
+        businessRuleTask.getSingleExtensionElement(ZeebeCalledDecision.class);
+
+    // then
+    assertThat(calledDecision.getBindingType()).isEqualTo(ZeebeBindingType.deployment);
+  }
+
+  @Test
+  public void shouldThrowExceptionForInvalidBindingType() {
+    // given
+    final BpmnModelInstance modelInstance =
+        Bpmn.readModelFromStream(getClass().getResourceAsStream("ZeebeCalledDecisionTest.bpmn"));
+    final BusinessRuleTask businessRuleTask =
+        modelInstance.getModelElementById("business-rule-task-2");
+
+    // when
+    final ZeebeCalledDecision calledDecision =
+        businessRuleTask.getSingleExtensionElement(ZeebeCalledDecision.class);
+
+    // then
+    assertThatThrownBy(calledDecision::getBindingType)
+        .isInstanceOf(IllegalArgumentException.class)
+        .hasMessage(
+            "No enum constant io.camunda.zeebe.model.bpmn.instance.zeebe.ZeebeBindingType.foo");
```

Comment:
💭 Maybe it would make sense to add a validation rule to return a more user-friendly error if an invalid binding type was used (something like `Attribute 'bindingType' must be one of: deployment, latest`). But I would add this in a follow-up PR.

Reply:
👍 I like this idea, and would suggest we take this along while this is still fresh. Fine to do so in a separate PR though.

Reply:
I created a [follow-up issue](https://github.com/camunda/camunda/issues/20054) for this and will tackle it soon.